SignalMessaging/utils/OWSWindowManager.m

```diff
@@ -43,13 +42,40 @@ const UIWindowLevel UIWindowLevel_CallView(void)
     return UIWindowLevelNormal + 1.f;
 }
 
-// In front of everything, including the status bar.
+// In front of the status bar and CallView
 const UIWindowLevel UIWindowLevel_ScreenBlocking(void);
 const UIWindowLevel UIWindowLevel_ScreenBlocking(void)
 {
     return UIWindowLevelStatusBar + 2.f;
 }
 
+// In front of everything
+const UIWindowLevel UIWindowLevel_MessageActions(void);
+const UIWindowLevel UIWindowLevel_MessageActions(void)
+{
+    // Note: To cover the keyboard, this is higher than the ScreenBlocking level,
+    // but this window is hidden when screen protection is shown.
+    return CGFLOAT_MAX;
```

Comment:
Wow, so the highest level is necessary?  Couldn't we just `UIWindowLevel_ScreenBlocking() + 1`?

Reply:
I guess I'm just worried that UIKit might not handle such high levels correctly.  Let's do the `-100` if you don't mind.

Reply:
Done!